src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -58,68 +58,76 @@ export default function useBulkImportFields() {
   const intl = useIntl();
   const { regionOptions, speciesOptions } = useOptions();
   const sightingFieldSchemas = useSightingFieldSchemas();
-  const flatfileSightingFields = useMemo(() => {
-    if (!sightingFieldSchemas) return {};
-    const bulkSightingFields = sightingFieldSchemas.filter(
-      f => !sightingOmitList.includes(f.name),
-    );
-    return bulkSightingFields.map(f => {
-      const additionalProperties = {};
-      if (
-        f?.fieldType === 'select' ||
-        f?.fieldType === 'multiselect'
-      ) {
-        additionalProperties.type = 'select';
-      }
-      if (f?.choices) additionalProperties.options = f.choices;
-      if (f.name === 'locationId') {
-        additionalProperties.type = 'select';
-        additionalProperties.options = regionOptions;
-        additionalProperties.validators = [requiredValidator];
-      }
-      return {
-        label: deriveLabel(f, intl),
-        key: deriveKey(f, categoryTypes.sighting),
-        ...additionalProperties,
-      };
-    });
-  }, [intl, sightingFieldSchemas, regionOptions]);
+  const flatfileSightingFields = useMemo(
+    () => {
+      if (!sightingFieldSchemas) return {};
+      const bulkSightingFields = sightingFieldSchemas.filter(
+        f => !sightingOmitList.includes(f.name),
+      );
+      return bulkSightingFields.map(f => {
+        const additionalProperties = {};
+        additionalProperties.customField = Boolean(f?.customField);
+        additionalProperties.fieldType = f?.fieldType;
+        if (f?.fieldType === 'select') {
+          additionalProperties.type = 'select';
+        }
+        if (f?.fieldType === 'multiselect')
+          additionalProperties.type = 'string';
```

Comment:
For Flatfile, "string" is not a valid type.
https://flatfile.com/developers/javascript/fields/#type